.github/release.yml

```
@@ -0,0 +1,17 @@
+changelog:
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - kind/breaking
+    - title: New Features 🎉
```

```suggestion
    - title: Enhancements 🎉
```

Following Traefik releases (e.g., https://github.com/traefik/traefik/releases/tag/v2.10.0-rc1) this section would be better named `Enhancements`.

---

Agreed. fixed in last commit